ci_group/revolve2/ci_group/morphological_novelty_metric/_morphological_novelty_metric.py

```diff
@@ -0,0 +1,124 @@
+from dataclasses import dataclass
+from math import atan2, pi, sqrt
+
+import numpy as np
+from numpy.typing import NDArray
+
+from revolve2.modular_robot import ModularRobot
+
+from ._coordinate_operations import coords_from_bodies
+from .calculate_novelty import calculate_novelty
+
+
+@dataclass(init=False)
+class MorphologicalNoveltyMetric:
```

Comment:
this class should also be a function right? if you really want to use a class like this, you should make the class private and add a function next to that does something like MorphoMetric().get_novelty(...)
so the user doesn't have to know there is a class. they never use the class for anything else than calling that single function so a class is an incorrect interface

Reply:
i made it a function now :) somehow i cant get around a del statement. setting the variables to None still produces some errors with nd arrays. I will investigate this further.